chess/backend/src/game/logic.py

```diff
@@ -2,6 +2,7 @@
 
 from django.contrib.auth.models import User
 from django.core.handlers.wsgi import WSGIRequest
+from django.db.models import Q
```

Comment:
What is Q?

Reply:
A django way to specify queries. You need to use a `Q` object to do `OR` and `AND` statements in your query